src/status_im/ui/screens/accounts/login/models.cljs

```diff
@@ -34,22 +37,26 @@
 
 ;;;; Handlers
 
-(defn open-login [address photo-path name {db :db}]
+(defn navigate-to-login [address photo-path name password {db :db}]
   {:db       (update db
                      :accounts/login assoc
                      :address address
                      :photo-path photo-path
+                     :password password
                      :name name)
    :dispatch [:navigate-to :login]})
```

Comment:
Minor point, just advice for the future.
We already have `navigate-to` exposed as pure function in navigation ns, so the above could be written like that without any additional dispatch (given that you already required navigation ns as `navigation`):
```clj
{:db (-> db
        (update :accounts/login merge {:address      address
                                       :photo-path photo-path
                                       :password   password
                                        :name         name})
        (navigation/navigate-to :login))}
```